omod/src/main/webapp/pages/editSection.gsp

```diff
@@ -10,8 +10,11 @@
 
     ui.includeCss("registrationapp", "editSection.css")
 
-    def genderOptions = [ [label: ui.message("emr.gender.M"), value: 'M'],
-                          [label: ui.message("emr.gender.F"), value: 'F'] ]
+    def translatedGenderOptions = []
```

Comment:
Keep the original name (`genderOptions`), in fact at the view level everything is supposed to be localized.

Reply:
Ok.